tests/test_ops/test_points_in_polygons.py

```diff
@@ -21,3 +22,35 @@ def test_points_in_polygons():
     expected_output = torch.from_numpy(expected_output).cuda().float()
     assert torch.allclose(
         points_in_polygons(points, polygons), expected_output, 1e-3)
+
+
+@pytest.mark.parametrize('device', [
+    pytest.param(
+        'cuda',
+        marks=pytest.mark.skipif(
+            not IS_CUDA_AVAILABLE, reason='requires CUDA support')),
+    pytest.param(
+        'mlu',
+        marks=pytest.mark.skipif(
+            not IS_MLU_AVAILABLE, reason='requires MLU support')),
+    pytest.param(
+        'npu',
+        marks=pytest.mark.skipif(
+            not IS_NPU_AVAILABLE, reason='requires NPU support'))
+])
+def test_points_in_polygons_device(device):
```

Comment:
```suggestion
def test_points_in_polygons(device):
```

Reply:
This test (https://github.com/open-mmlab/mmcv/pull/2848/files#diff-efebcc8384b4e339d2672c04d405af66df075e5e231e29bab2571442cb0545d4R12) can be removed.

Reply:
modified according to suggestions